deploy/olm-catalog/ibm-ingress-nginx-operator/1.11.0/ibm-ingress-nginx-operator.v1.11.0.clusterserviceversion.yaml

```diff
@@ -123,7 +123,7 @@ spec:
       image: quay.io/opencloudio/default-http-backend:1.5.9
     - name: KUBECTL_IMAGE
       image: quay.io/opencloudio/kubectl:v1.15.9.6
-    - name: ibm-ingress-nginx-operator
+    - name: IBM_INGRESS_NGINX_OPERATOR_IMAGE
```

Comment:
shouldn't the `:tag` be used, can you please double check, after CICD executes, whther the daaily build would contain the actual SHA digest in the built CSV?

Reply:
Added the tag. Thank you, @pgodowski